modules/renter/download.go

```diff
@@ -233,6 +233,105 @@ func (d *download) Err() (err error) {
 	return err
 }
 
+// Download performs a file download using the passed parameters and blocks
+// until the download is finished.
+func (r *Renter) Download(p modules.RenterDownloadParameters) error {
+	return r.managedDownload(p, false)
```

Comment:
```go
err := r.managedDownload(p)
if err != nil {
    return err
}

select {
    // Block until download is complete
}
return d.Err()
```

That way you don't need the true/false stuff in your function.

Reply:
How should we wait for the download without returning a channel?